### PR TITLE
Google Analytics Added

### PR DESCRIPTION
**Pull request by abhishek-bodapati**
Resolved issue #47 

- Google Measurement ID be kept in ```GA_MEASUREMENT_ID``` in ```.env```
- Used vue-gtag package
- Added the script in ```resources/views/spa.blade.php```
- The script runs only if the MEASUREMENT_ID is available. Hopefully, this won't disturb the working of the project.
- Tested in my local website using ngrok (to tunnel local host to public URL)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,18 +17,21 @@
     "axios": "^0.21.1",
```
Comment (Zelig880) [inline package.json:17]: Please revert the changes to this file, i do not think you need to change the package.json and also the package-lock.json with your changes :)

File: resources/views/spa.blade.php [hunk 2]
```diff
@@ -24,13 +26,31 @@
...
 
+	<script>
+    var m_id = '<?php echo $measurement_id; ?>';
+    var src_url = 'https://www.googletagmanager.com/gtag/js?id=' + m_id; // Appending m_id to URL
+	//alert(src_url);
+	</script>
+
+	<!-- Global site tag (gtag.js) - Google Analytics -->
+	<?php 
+	if($measurement_id != 'None') {
```
Comment (Zelig880) [inline resources/views/spa.blade.php:37]: it is always suggested to use `!==` than `!=` as php like javascript does not check its type if you just use 1 equal

File: resources/views/spa.blade.php [hunk 3]
```diff
@@ -24,13 +26,31 @@
...
+	  echo "<script async src=src_url></script>
+	        <script>
+	          window.dataLayer = window.dataLayer || [];
+	          function gtag(){dataLayer.push(arguments);}
+	          gtag('js', new Date());
+	          gtag('config', m_id);
+	        </script>";
+	}
+	?>
```
Comment (Zelig880) [inline resources/views/spa.blade.php:46]: just to let you know, instead than declare the PHP block like you did and "echo" html, in the PHP blade you can use a much cleaner syntax like explained in this link: https://stackoverflow.com/questions/40122633/if-else-condition-in-blade-file-laravel-5-3

@if($measurement_id !== 'None')         
      <script> your script tag</script>    
@endif

**Review by Zelig880 — CHANGES_REQUESTED** [review]
The PR was very good.. I have just provided some comment to make sure the code is consistent with the rest..

1) Remove the extra file pushed
2) Remove the changes to package.json and Package-lock.json
3) Improve the way you call PHP method and file on the blade (worth reading about laravel blade, as you can do so much more and cleaner (you can use {{ }}, or @ for variables, withouth the need to open php block like you were doing with <?php ?>

**Comment by abhishek-bodapati** [comment]
Sure !! I'll do the necessary changes.

**Comment by Zelig880** [comment]
This has already been merged in another patch

**Pull request closed without merging**